src/SearchBar.tsx

```diff
@@ -10,31 +11,70 @@ interface SearchBarProps {
 }
 
 export default function SearchBar({ currentGuess, setCurrentGuess, guessList, setGuessList }: SearchBarProps) {
-    const [searchInput, setSearchInput] = useState<String>("")
+    const [searchOptions, setSearchOptions] = useState<MovieBite[]>([])
+    const [selectedMovie, setSelectedMovie] = useState<MovieBite>({id: NaN, title: "", year: ""})
     const toast = useToast()
-    
-    const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
-        setSearchInput(e.target.value)
-    }
 
-    function searchByInput() {
-        let optionsURL: string = "http://localhost:5050/getMovieOptions?title=" + searchInput
+    const handleInputChange = (input: string) => {
+        if (input === "") {
+            setSelectedMovie({id: NaN, title: "", year: ""})
+            return
+        }
+        let optionsURL: string = "http://localhost:5050/getMovieOptions?title=" + input
```

Comment:
we don't want to hardcode urls like this, use environment variables to get localhost:5050, you can leave the rest.

Reply:
Leaving this in for now - will change in next PR where all pointers will move from localhost to AWS. Noted about hardcoding URL, though